docs/csharp/language-reference/builtin-types/struct.md

```diff
@@ -0,0 +1,65 @@
+
+[!code-csharp[struct example](~/samples/csharp/language-reference/builtin-types/StructType.cs#StructExample)]
+
+Structure types have *value semantics*. That is, a variable of a structure type contains an instance of the type. By default, variable values are copied on assignment, passing an argument to a method, and returning a method result. In the case of a structure-type variable, an instance of the type is copied. For more information, see [Value types](value-types.md).
+
+Typically, you use structure types to design small data-centric types that provide little or no behavior. For example, .NET uses structure types to represent a number (both [integer](integral-numeric-types.md) and [real](floating-point-numeric-types.md)), a [Boolean value](bool.md), a [Unicode character](char.md), a [time instance](xref:System.DateTime). If you're focused on the behavior of a type, consider defining a [class](../keywords/class.md). Class types have *reference semantics*. That is, a variable of a class type contains a reference to an instance of the type, not the instance itself.
+
+## Limitations with the design of a structure type
+
+When you design a structure type, you have the same capabilities as with a [class](../keywords/class.md) type, with the following exceptions:
+
+- You cannot declare a parameterless constructor. Every structure type already provides an implicit parameterless constructor that produces the [default value](default-values.md) of the type.
+
+- You cannot initialize an instance field or property at its declaration. However, you can initialize a [static](../keywords/static.md) or [const](../keywords/const.md) field or a static property at its declaration.
+
+- A constructor of a structure type must initialize all instance fields of the type.
+
+- A structure type cannot inherit from other class or structure type and it cannot be the base of a class. However, a structure type can implement [interfaces](../keywords/interface.md).
+
+- You cannot declare a [finalizer](../../programming-guide/classes-and-structs/destructors.md) within a structure type.
+
+## Instantiation of a structure type
+
+In C#, you must initialize a declared variable before it can be used. Because a structure-type variable cannot be `null` (unless it's a variable of a [nullable value type](nullable-value-types.md)), you must instantiate an instance of the corresponding type. There are several ways to do that.
+
+Typically, you instantiate a structure type by calling an appropriate constructor with the [`new`](../operators/new-operator.md) operator. Every structure type has at least one constructor. That's an implicit parameterless constructor, which produces the [default value](default-values.md) of the type. You can also use the [default](../operators/default.md) operator or literal to produce the default value of a type.
```

Comment:
As it's a general recommendation to define *immutable* value types, I haven't mentioned object initializers here. However, the linked article about the `new` operator discusses object initializers.

Reply:
> Have you been reading the discussions about records and initializations for future versions?

I've noticed them and looked through but didn't read in details and thoughtfully yet.